profile/README.md

```diff
@@ -1,5 +1,3 @@
-## Greptime builds Cloud-Native TSDB Services: Anywhere, Any Scale
-
 <p align="center">
   <picture>
     <source media="(prefers-color-scheme: light)" srcset="/img/logo/text-padding-tined.svg">
```

Comment:
Let's change the logo too.

Reply:
Where is the logo asse, or it's under design?

Reply:
See https://greptime.com/logo-resource

Reply:
Updated.